copy-to-clipboard/index.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+let templateHtml = require('./copy-to-clipboard.html');

```

Comment:
this doesn't work with node's `require`
we would have to include `stringify` as a dependency

also... where is _copy-to-clipboard.html_?